docker-compose.yml

```
@@ -57,6 +57,13 @@ services:
       - MONGO_INITDB_ROOT_USERNAME=root
       - MONGO_INITDB_ROOT_PASSWORD=root-password
 
+  mongonoauth:
+    image: ${MONGO_IMAGE:-percona/percona-server-mongodb:4.2}
+    container_name: pmm-agent_mongo_noauth
```

```suggestion
    container_name: pmm-agent_mongonoauth
```

Or `mongo_noauth` two lines above. For consistency.